### PR TITLE
Annotate all types of mutations with cbioportal and cosmic

### PR DESCRIPTION
**Pull request by adamabeshouse**

### CODE REVIEW

**Comment by alisman** [comment]
@jjgao this PR needs tests. Can it wait until @adamabeshouse comes back?

**Comment by jjgao** [comment]
@alisman yes.

File: src/pages/resultsView/ResultsViewPageStore.ts [hunk 1]
```diff
@@ -1654,8 +1654,7 @@ export class ResultsViewPageStore {
         invoke: ()=>{
             return internalClient.fetchCosmicCountsUsingPOST({
                 keywords: _.uniq(this.mutations.result!.filter((m:Mutation)=>{
-                    const simplifiedMutationType = getSimplifiedMutationType(m.mutationType);
-                    return (simplifiedMutationType === "missense" || simplifiedMutationType === "inframe") && !!m.keyword;
+                    return !!m.keyword;
```
Comment (alisman) [inline src/pages/resultsView/ResultsViewPageStore.ts]: would love a comment here explaining why we want to filter for items with a keyword.

File: src/shared/components/oncoprint/ResultsViewOncoprint.tsx [hunk 2]
```diff
@@ -313,6 +313,10 @@ export default class ResultsViewOncoprint extends React.Component<IResultsViewOn
                 }
             },
         });
+
+        // expose functions for testing
+        (window as any).__onChangeAnnotateCBioPortalInputValue = this.controlsHandlers.onChangeAnnotateCBioPortalInputValue;
```
Comment (alisman) [inline src/shared/components/oncoprint/ResultsViewOncoprint.tsx]: ditto

**Pull request merged**